Mathlib/SetTheory/Cardinal/Cofinality.lean

```diff
@@ -1248,3 +1248,20 @@ theorem lt_cof_power {a b : Cardinal} (ha : ℵ₀ ≤ a) (b1 : 1 < b) : a < cof
 #align cardinal.lt_cof_power Cardinal.lt_cof_power
 
 end Cardinal
+
+section Omega1
```

Comment:
It may feel a bit redundant to enclose a `namespace ... end` with a `section`, but maybe it's a good way to organize things. (Personally I might just add a comment rather than using a section; sections can be collapsed in VSCode but you may as well collapse the namespace.)

Reply:
The style I have seen is having a section for each new concept, I was only trying to mimic this. As always, I open to modification if it is better for mathlib.